### ISSUE TITLE
"could not reach the target" while using coordinates with valetudo re v0.10.8

### ISSUE THREAD

**Issue by eldrik** [rand256/valetudo#495]
Hi, i tried to use the new feature using coordintes.

I gathered the coordinates via the valetudo re map and published a mqtt message them via valetudo/rockrobo/custom_command {"command":"go_to","spot_coordinates": {"x":57850,"y":22450}}

The command is recognized

{"command":"go_to","message":["ok"],"error":null,"updated":1665828990967}

 and the robo is leaving the dock, but after 1-2 seconds it responds with could not reach the target :/

Any tips what might cause this error? 

Regards
Eldrik

**Comment by eldrik** [comment]
ok, figured it out. 

The coordinates that I figured out via the map feature are not the right ones, with those coordinates the target was outside of the map.
Figured the coordinates out by try and error and they are instead of 

{"x": 57850, "y": 22450}}

{"x": 23550, "y": 21050}}

Is there a problem with the map feature or is maybe a full new clean run needed? I am working with saved maps for different clean runs on the ground and upper floor.

Regards
Eldrik
